lib/gooddata/lcm/types/special/any.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  module LCM2
+    module Type
+      class AnyType
+        CATEGORY = :special
+        def check(values)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `values`. If it's necessary, use `_` or `_values` as an argument name to indicate that it won't be used. You can also write as `check(*)` if you want the method to accept any arguments but don't care about them.